### ISSUE TITLE
Show Up-to-date and Out-dated docker containers

### ISSUE THREAD

**Issue by gordnhoo** [IceWhaleTech/CasaOS#1331]
I would like to have a green and red icon on the top right "or whatever" of every container.

This would be the same has what portainer does in their Business Edition.

Below is a screenshot of what it could look like.
https://prnt.sc/_rqQ4z-e1-aB

**Comment by CorrectRoadH** [comment]
Good idea @ETWang1991 what do you think about it? can we move it to our roadmap?

**Comment by AmadeusGraves** [comment]
This has to be coordinated with this issue [#1330](https://github.com/IceWhaleTech/CasaOS/issues/1330#issuecomment-1668907037)

**Comment by WisdomSky** [comment]
I've made a PR which adds showing a badge if an update is available so users will be notified. 

It's up to the team if they'll approve this change.

https://github.com/IceWhaleTech/CasaOS-UI/pull/217

**Comment by gordnhoo** [comment]
> I've made a PR which adds showing a badge if an update is available so users will be notified. 
> 
> It's up to the team if they'll approve this change.
> 
> https://github.com/IceWhaleTech/CasaOS-UI/pull/217

Seems good enough I really wanted this feature because it helps even if you have a tool like watchtower sometimes it can fail for some reason so this is good thank you a lot changed to casa os a few months ago and have been loving it

**Comment by WisdomSky** [comment]
The PR was already merged, now we'll just have to wait for the next CasaOS update/release.

**Comment by ETWang1991** [comment]
That sounds like a good idea to note, but I'm wondering if there's a more elegant way to do it

**Comment by gordnhoo** [comment]
> elegant

Hmm I like the single dot but there could be a button for when an update is available or simply a red dot when the package is not up to date but I think we could also have the option to add the button or dot if we want this way not everyone is obliged to have the dot or update button on their screen

**Comment by zhanghengxin** [comment]
New feedback: When the application version is too high, higher than the version in the CasaOS app store, the update flag appears.
